.github/CONTRIBUTING.md

```diff
@@ -159,8 +159,7 @@ Testing your work locally will help you speed up the process since it allows you
 To setup a local development environment, install both local and test dependencies:
 
 ```bash
-python -m pip install -r requirements/devel.txt
-python -m pip install -r requirements/examples.txt
+python -m pip install ".[dev, examples]"
```

Comment:
maybe devel to match the file name?
```suggestion
python -m pip install ".[devel, examples]"
```

Reply:
I think dev is a better name, and since the user doesn't have to use interface with the reqs files now, I'd rather stick with dev. Does that sound reasonable? Totally see your point too.